doc/manual/mathematical-operations.rst

```diff
@@ -454,6 +454,7 @@ Function                     Description
 :func:`cld(x,y) <cld>`       ceiling division; quotient rounded towards ``+Inf``
 :func:`rem(x,y) <rem>`       remainder; satisfies ``x == div(x,y)*y + rem(x,y)``; sign matches ``x``
 :func:`mod(x,y) <mod>`       modulus; satisfies ``x == fld(x,y)*y + mod(x,y)``; sign matches ``y``
+:func:`mod1(x,y) <mod1>`     modulus with offset 1; gives a result ``r`` such that ``mod(r, y)==mod(x,y)``; sign matches ``y``
```

Comment:
The documentation of `mod1` should use the function name `mod1` as well.

`mod(r, y) == mod(x,y)` is a post-condition, but is not sufficient to unambiguously define the result. I believe there is a good definition elsewhere. Here is what the source code says:
```Julia
# mod returns in [0,y) or (y,0] (for negative y),
# whereas mod1 returns in (0,y] or [y,0)
```